target/linux/ipq806x/patches-4.19/0052-PM-OPP-Update-the-voltage-tolerance-when-adjusting-t.patch

```diff
@@ -19,20 +19,20 @@ Signed-off-by: Georgi Djakov <georgi.djakov@linaro.org>
 +++ b/drivers/opp/core.c
```

Comment:
This commit subject and description needs more love.

Reply:
Good now?

Reply:
I've meant this one ![image](https://user-images.githubusercontent.com/43848/71447903-24100100-2735-11ea-8748-bd92777a1c73.png) as it's not possible to comment on it...

Reply:
Sorry for the stupid question... How should i change them ?

Reply:
Imagine, that you're reading the Git history somewhere in 2021 and see `ipq806x: update 0052`, would you be able to tell from this commit subject what is this commit about?

Reply:
And this patch is probably some upstream backport, so perhaps your changes should be upstreamed first as well?

Reply:
Thx for the explanation. 
About proposing the patch upstream i can try... Anyway i have updated it for now...